### PR TITLE
Remove stage presets

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
babel/babel#8293

### CODE REVIEW

File: src/packageData.js [hunk 1]
```diff
@@ -245,9 +245,39 @@ const packages = Object.assign(
...
+  "@babel/plugin-syntax-dynamic-import",
+  "@babel/plugin-syntax-import-meta",
+  "@babel/plugin-proposal-class-properties",
+  "@babel/plugin-proposal-json-strings",
+];
+stagePresets[2] = [
+  ...stagePresets[3],
+  ["@babel/plugin-proposal-decorators", { "legacy": true }],
+  "@babel/plugin-proposal-function-sent",
+  "@babel/plugin-proposal-export-namespace-from",
```
Comment (hzoo) [inline src/packageData.js]: FYI This got moved to a needs consensus but not merged yet: https://github.com/tc39/ecma262/pull/1174

**Comment by hzoo** [comment]
Going to be interesting to maintain, hopefully it's not that bad.

**Comment by nicolo-ribaudo** [comment]
I don't think that we should update those lists, since they only represents what stage presets had before being removed.

**Comment by hzoo** [comment]
Yeah that's what's I was getting at as well, 👍

**Pull request merged**